packages/components/src/Input/Input/Input.module.scss

```diff
@@ -357,3 +356,4 @@ $input-with-icon-padding: calc(
     }
   }
 }
+/* stylelint-enable no-descending-specificity */
```

Comment:
The styles are too messy to quickly fix, so opted to just ignore it. Better we do a proper refactor later.